web/docs/project/starter-templates.md

```diff
@@ -82,8 +84,23 @@ wasp new <project-name> -t todo-ts
 ### AI Generated Starter 🤖
 
 Using the same tech as used on https://usemage.ai/, Wasp generates your custom starter template based on your
-project description. It will automatically generate your data model, auth, queries, actions and React pages. 
+project description. It will automatically generate your data model, auth, queries, actions and React pages.
 
 _You will need to provide your own OpenAI API key to be able to use this template._
 
-**Features:** Generated using OpenAI's GPT models, Auth (username/password), Queries, Actions, Pages, Full-stack Type Safety
\ No newline at end of file
+**Features:** Generated using OpenAI's GPT models, Auth (username/password), Queries, Actions, Pages, Full-stack Type Safety
+
+
+## Community templates #{community-templates}
```

Comment:
This id in #{} got rendered in docs! See what is with that.

Reply:
What do you mean? How does it look like?